src/mmw/apps/home/views.py

```diff
@@ -42,9 +44,12 @@ def model(request, proj_id=None):
     csrf_token.update(csrf(request))
 
     if proj_id:
-        get_object_or_404(Project, user=request.user.id, id=proj_id)
+        project = get_object_or_404(Project, id=proj_id)
 
-    return render_to_response('home/home.html', csrf_token)
+    if project.user.id == request.user.id or project.is_private is False:
```

Comment:
In this case, I think `not project.is_private` is correct. If `is_private` was nullable, you would need to use the `is` operator (if you wanted to distinguish null from `False`).